common/src/main/java/com/wynntils/models/items/annotators/game/HorseAnnotator.java

```diff
@@ -29,7 +29,30 @@ public ItemAnnotation getAnnotation(ItemStack itemStack, StyledText name) {
         if (!matcher.matches()) return null;
 
         Matcher tierMatcher = LoreUtils.matchLoreLine(itemStack, 0, HORSE_TIER_PATTERN);
-        if (!tierMatcher.matches()) return null;
+
+        // The lore may be missing if the item is on the trade market
+        if (!tierMatcher.matches()) {
+            int tier;
+
+            switch (matcher.group(1)) {
```

Comment:
At least there is a HorseModel. :-) Create a new HorseTier inner enum in it, move the string literals and corresponding tier numbers there, and add a method for converting from string to tier number to HorseModel.

Reply:
If I'm going to do this I might as well add models for the other things too.